infra/terraform/dev/deploy/iam.tf

```diff
@@ -0,0 +1,149 @@
+#####
+# Execution IAM Role
+#####
+resource "aws_iam_role" "execution" {
+  name               = "wj-${var.environment}-execution-role"
+  assume_role_policy = data.aws_iam_policy_document.assume_role_policy.json
+}
+
+resource "aws_iam_role_policy_attachment" "ecs_task_execution_role_policy_attach" {
+  role       = aws_iam_role.execution.name
+  policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy"
```

Comment:
Why does this use a hardcoded string and this next resource uses a variable?

Reply:
This is a known value from AWS, a globally available policy that has the exact permission set needed for this piece.